syntax = "proto3";

package response_models;
import "vector2d.proto";


message PlayerPositionResponseModel {
	int32 playerId = 1;
	util_models.Vector2D position = 2;
	util_models.Vector2D velocity = 3;
}
